src/wp-includes/l10n.php

```diff
@@ -725,12 +725,14 @@ function load_textdomain( $domain, $mofile, $locale = null ) {
 	 * Filters whether to override the .mo file loading.
 	 *
 	 * @since 2.9.0
+	 * @since 6.2.0 Added the `$locale` parameter.
 	 *
 	 * @param bool   $override Whether to override the .mo file loading. Default false.
 	 * @param string $domain   Text domain. Unique identifier for retrieving translated strings.
 	 * @param string $mofile   Path to the MO file.
+	 * @param string $locale   Locale.
```

Comment:
```suggestion
	 * @param string|null $locale   Locale.
```

We should document that $locale can be null since that's the default value. This is also an oversight in the `@param` doc for the function. Might updating it there as well?